pkg/master/reconcilers/endpointsadapter.go

```diff
@@ -112,7 +112,7 @@ func endpointSliceFromEndpoints(endpoints *corev1.Endpoints) *discovery.Endpoint
 	endpointSlice.Name = endpoints.Name
 	endpointSlice.Labels = map[string]string{discovery.LabelServiceName: endpoints.Name}
 
-	ipAddressType := discovery.AddressTypeIP
+	ipAddressType := discovery.AddressTypeIPv4
```

Comment:
How do you know?

Reply:
It doesn't look like these reconcilers have any dual stack support at this point, and adding it felt out of scope. I've added a todo comment for now, and reached out to @khenidak to see if there are already any dual stack related changes on the way here.